src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs

```diff
@@ -13,9 +13,25 @@ public static class LinkedInAuthenticationConstants
 {
     public static class Claims
     {
+        [Obsolete("PictureUrls is deprecated, no longer conforms to data returned by recent API, please use Picture instead.", false)]
```

Comment:
Please change these messages to something like:

```
$"This constant is obsolete and is no longer used. It will be removed in a future version. Use {nameof(ReplacementName)} instead."
```